ecosystem/typescript/sdk/src/tests/unit/test_helper.test.ts

```diff
@@ -30,4 +32,4 @@ test("noop", () => {
   // 2. Prevent jest from exploding when it finds a file with no tests in it
 });
 
-export const longTestTimeout = 80 * 1000;
+export const longTestTimeout = 120 * 1000;
```

Comment:
can we later try to make the tests finish in a shorter time? otherwise we keep increasing this timeout lol

Reply:
Yeah, 2 minutes is relatively long, we should look into what we can do to make this better handled.

Reply:
yes.... I have a todo for that